### ISSUE TITLE
Not compatible with symfony 2.8?

### ISSUE THREAD

**Issue by falsecrypt** [alexdebril/rss-atom-bundle#124]
Hi,
can not install the latest version of this bundle with symfony 2.8. Composer says i should remove symfony v2.8. Is this bundle only compatible with >= symfony 3.0? I could not find that in the documentation.

```
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove symfony/symfony v2.8.14
    - Conclusion: don't install symfony/symfony v2.8.14
    - Conclusion: don't install symfony/symfony v2.8.13
    - Conclusion: don't install symfony/symfony v2.8.12
    - Conclusion: don't install symfony/symfony v2.8.11
    - Conclusion: don't install symfony/symfony v2.8.10
    - Conclusion: don't install symfony/symfony v2.8.9
    - Conclusion: don't install symfony/symfony v2.8.8
    - Conclusion: don't install symfony/symfony v2.8.7
    - Conclusion: don't install symfony/symfony v2.8.6
    - Conclusion: don't install symfony/symfony v2.8.5
    - Conclusion: don't install symfony/symfony v2.8.4
    - Conclusion: don't install symfony/symfony v2.8.3
    - Installation request for debril/rss-atom-bundle ^3.0 -> satisfiable by debril/rss-atom-bundle[v3.0.0].
    - Conclusion: don't install symfony/symfony v2.8.2
    - Conclusion: don't install symfony/symfony v2.8.1
    - debril/rss-atom-bundle v3.0.0 requires debril/feed-io ^2.4 -> satisfiable by debril/feed-io[v2.4.0, v2.4.1, v2.4.2, v2.4.3].
    - debril/feed-io v2.4.0 requires symfony/console ^3.1 -> satisfiable by symfony/console[v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5, v3.1.6, v3.1.7].
    - debril/feed-io v2.4.1 requires symfony/console ^3.1 -> satisfiable by symfony/console[v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5, v3.1.6, v3.1.7].
    - debril/feed-io v2.4.2 requires symfony/console ^3.1 -> satisfiable by symfony/console[v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5, v3.1.6, v3.1.7].
    - debril/feed-io v2.4.3 requires symfony/console ^3.1 -> satisfiable by symfony/console[v3.1.0, v3.1.1, v3.1.2, v3.1.3, v3.1.4, v3.1.5, v3.1.6, v3.1.7].
    - don't install symfony/console v3.1.0|don't install symfony/symfony v2.8.0
    - don't install symfony/console v3.1.1|don't install symfony/symfony v2.8.0
    - don't install symfony/console v3.1.2|don't install symfony/symfony v2.8.0
    - don't install symfony/console v3.1.3|don't install symfony/symfony v2.8.0
    - don't install symfony/console v3.1.4|don't install symfony/symfony v2.8.0
    - don't install symfony/console v3.1.5|don't install symfony/symfony v2.8.0
    - don't install symfony/console v3.1.6|don't install symfony/symfony v2.8.0
    - don't install symfony/console v3.1.7|don't install symfony/symfony v2.8.0
    - Installation request for symfony/symfony 2.8.* -> satisfiable by symfony/symfony[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].

```

**Comment by alexdebril** [comment]
Hi, 

This isn't normal, I think composer.json is wrong on dependencies' versions. This must be fixed quickly, I'll release a new version in the next few days. 

Thank you for your feedback 

Alex

**Comment by alexdebril** [comment]
Hi again,

here's a workaround : 

```
composer require debril/rss-atom-bundle ^2.4
```

That's because feed-io's not compatible with symfony/console v2.8, it will be easy to fix.

**Comment by alexdebril** [comment]
Now it's fixed, I tested with a Symfony 2.8 stack

Thank you,

Alex
